js/functions.js

```diff
@@ -536,57 +561,57 @@ function loadActualStartTime() {
 }
 
 function loadAircrafts(callback) {
-    $.getJSON("data/aircrafts-info.json?t=" + (new Date()).getTime(), function (aircraftInfo) {
-        // load aircraft type info into a map
-        aircraftInfo.aircraftTypes.forEach(function (aircraftTypeInfo) {
-            aircraftTypesInfo[aircraftTypeInfo.aircraftTypeId] = aircraftTypeInfo;
-        }, this);
+    getEnv((env) => {
```

Comment:
why do we need to getEnv every time? it should happen only once and then be stored in a common variable that can be used by anyone.